pygmo/core.cpp

```diff
@@ -245,6 +245,17 @@ static inline void population_prob_init(bp::class_<population> &pop_class)
         .def(bp::init<const Prob &, population::size_type, unsigned>());
 }
 
+// champion_x()
+static inline bp::object pop_champion_x_wrapper(population &pop)
+{
+    return pygmo::v_to_a(pop.champion_x());
+}
+// champion_f()
+static inline bp::object pop_champion_f_wrapper(population &pop)
```

Comment:
Instead of writing these wrappers, could you just put them in the ``.add_property()`` call below with the usual ``+[]`` lambda trick?